pkg/common/x509svid/upstreamca.go

```diff
@@ -84,7 +80,7 @@ func (ca *UpstreamCA) SignCSR(ctx context.Context, csrDER []byte) (*x509.Certifi
 			x509.KeyUsageCertSign |
 			x509.KeyUsageCRLSign,
 		BasicConstraintsValid: true,
-		IsCA:                  true,
+		IsCA: true,
```

Comment:
Is this a go fmt change that got reverted somehow?

Reply:
Hmm... :thinking: It looks like when go fmt is called from my IDE on save, this line gets changed... I'll revert it.